test-runner.js

```diff
@@ -1,5 +1,9 @@
+require('babel-polyfill');
+
 const testsContext = require.context('./tests', true, /\.js$/);
 const componentsContext = require.context('./src/', true, /components\/.*\.js$/);
+const reducersContext = require.context('./src/', true, /reducers\/.*\.js$/);
```

Comment:
Nice, I suppose the next logical progression to this would be to add actions too.